ios/Classes/SwiftGigyaFlutterPluginTyped.swift

```diff
@@ -63,6 +63,27 @@ public class SwiftGigyaFlutterPluginTyped<T: GigyaAccountProtocol> : NSObject, F
         case .getSession:
             sdk?.getSession(result: result)
             return
+        case .biometricIsAvailable:
+            sdk?.isAvailable(result: result)
```

Comment:
Maybe we can make the methods in the GigyaSdkWrapper also prefixed with `biometrics` ?

For example: if we were to add a method `isAvailable` for a _different_ thing, then we won't have conflicts?